frontend/src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,104 @@
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+
+import ThemeSwitch from './ThemeSwitch/ThemeSwitch';
+
+const pages = ['Daily Drugs', 'Lexicon', 'Calendar'];
+const LOGO_IMG = { path: '../../../public/logo_color.png', alt: 'logo' };
+
+const Navbar = () => {
+  const [anchorElNav, setAnchorElNav] = React.useState(null);
+
+  const handleOpenNavMenu = (event) => {
+    setAnchorElNav(event.currentTarget);
+  };
+
+  const handleCloseNavMenu = () => {
+    setAnchorElNav(null);
+  };
+
+  return (
+    <AppBar position="static">
+      <Container maxWidth="xl">
+        <Toolbar disableGutters>
+          <Avatar alt="logo" src={LOGO_IMG.path} sx={{ width: 50, height: 50, display: { xs: 'none', md: 'flex' } }} />
+          <Typography variant="h6" noWrap component="div" sx={{ mr: 2, ml: 2, display: { xs: 'none', md: 'flex' } }}>
+            aID kIT
+          </Typography>
+
+          <Box sx={{ flexGrow: 1, display: { xs: 'flex', md: 'none' } }}>
+            <IconButton
+              size="large"
+              aria-label="account of current user"
+              aria-controls="menu-appbar"
+              aria-haspopup="true"
+              onClick={handleOpenNavMenu}
+              color="inherit"
+            >
+              <MenuIcon />
+            </IconButton>
+            <Menu
+              id="menu-appbar"
+              anchorEl={anchorElNav}
+              anchorOrigin={{
+                vertical: 'bottom',
+                horizontal: 'left',
+              }}
+              keepMounted
+              transformOrigin={{
+                vertical: 'top',
+                horizontal: 'left',
+              }}
+              open={Boolean(anchorElNav)}
+              onClose={handleCloseNavMenu}
+              sx={{
+                display: { xs: 'block', md: 'none' },
+              }}
+            >
+              {pages.map((page) => (
+                <MenuItem key={page} onClick={handleCloseNavMenu}>
+                  <Typography textAlign="center">{page}</Typography>
+                </MenuItem>
+              ))}
+            </Menu>
+          </Box>
+          <Box sx={{ flexGrow: 1, display: { xs: 'flex', md: 'none' } }}>
+            <Avatar alt="logo" src={LOGO_IMG.path} sx={{ width: 50, height: 50 }} />
+            <Typography variant="h6" noWrap component="div" sx={{ ml: 2, mt: 1 }}>
+              aID kIT
+            </Typography>
+          </Box>
+
+          <Box sx={{ flexGrow: 1, display: { xs: 'none', md: 'flex' } }}>
+            {pages.map((page) => (
+              <Button key={page} onClick={handleCloseNavMenu} sx={{ my: 2, color: 'white', display: 'block' }}>
+                {page}
+              </Button>
+            ))}
+          </Box>
```

Comment:
Mobile and web version makes this component's return quite long. It may be good idea to extract those markups to the separate components (even within the same file) eg. MenuBoxMobile and MenuBoxWeb

Reply:
done